doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -608,6 +608,10 @@ Modem libraries
 
   * Removed support for deprecated RAI socket options ``SO_RAI_LAST``, ``SO_RAI_NO_DATA``, ``SO_RAI_ONE_RESP``, ``SO_RAI_ONGOING``, and ``SO_RAI_WAIT_MORE``.
 
+* :ref:`nrf_modem_trace` library
+
+  * The RTT trace backend now allocates the RTT channel at boot, instead of when the modem is activated.
```

Comment:
`nrf_modem_trace` references the modem trace API of the modem library in sdk-nrfxlib. The trace backends are part of the modem library integration:
```suggestion

  * Removed support for deprecated RAI socket options ``SO_RAI_LAST``, ``SO_RAI_NO_DATA``, ``SO_RAI_ONE_RESP``, ``SO_RAI_ONGOING``, and ``SO_RAI_WAIT_MORE``.
  * The RTT trace backend now allocates the RTT channel at boot, instead of when the modem is activated.
```